template/_gitignore

```diff
@@ -20,6 +20,7 @@ DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
+ios/.xcode.local.env
```

Comment:
I think in the js world it was established by CRA (https://create-react-app.dev/docs/adding-custom-environment-variables/) and others to use .env and .env.local. Maybe we should use the same here, so .xcode.env and .xcode.env.local?